server/api/command.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/mattermost/mattermost-plugin-solar-lottery/server/command"
+	"github.com/mattermost/mattermost-plugin-solar-lottery/server/utils/types"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+func (s *Service) executeCommand(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
+	if userID == "" {
+		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		return
+	}
```

Comment:
One post merge security question: This check ensure that user has to be logged in to trigger this API endpoint. Are there check that ensure that the user has access to `args.ChannelId`?